KubeArmor/BPF/common.h

```diff
@@ -0,0 +1,33 @@
+/* SPDX-License-Identifier: GPL-2.0 */
+/* Copyright 2024 Authors of KubeArmor */
+/* This module contains the common structures shared by lsm and system monitor*/
+# include "throttling.h"
+#ifndef __COMMON_H
+#define __COMMON_H
+#define MAX_ENTRIES 10240
+#define MAX_ARGUMENT_SIZE 256
+#define MAX_STR_ARR_ELEM 20
+
+// arguments matching 
+
+// values stored for argument map
+struct argVal{
```

Comment:
need to discuss on length and number of arguments that should be stored per command